docs/security/05-05-production-profile.md

```diff
@@ -90,9 +161,10 @@ You can deploy a Kyma cluster with the Hydra OAuth2 server configured to use the
         component: ory
         kyma-project.io/installation: ""
     data:
-      postgresql.enabled: "true"
+      global.ory.hydra.persistence.enabled: "true"
+      global.ory.hydra.persistence.postgresql.enabled: "true"
+      global.ory.hydra.persistence.gcloud.enabled: "false"
       hydra.hydra.autoMigrate: "true"
-      global.ory.hydra.persitance.enabled: "true"
     EOF
     ```
```

Comment:
Please add example for gcp sql which will be displayed as a 3rd tab/option as an example